lib/task_helpers/imports/roles.rb

```diff
@@ -0,0 +1,31 @@
+module TaskHelpers
+  class Imports
+    class Roles
+      def import(options = {})
+        return unless options[:source]
+
+        glob = File.file?(options[:source]) ? options[:source] : "#{options[:source]}/*.yaml"
+        Dir.glob(glob) do |fname|
+          begin
+            roles = YAML.load_file(fname)
+            import_roles(roles)
+          rescue => e
```

Comment:
Is there more specific exception we can rescue here?

Reply:
I don't know what all of the exceptions are that could be raised so I used the more generic to catch any error with the import.